web/blog/2022-05-31-filip-intro.md

```diff
@@ -0,0 +1,145 @@
+all the best practices that come with it.
+
+And then, one day, a friend of mine mentioned Wasp - a DSL aiming to eliminate
+boilerplate from web apps. The product was a perfect fit based on everything I
+know and want to do! Also, the founders (Martin and Matija) and I shared an
+advisor in college, and I've only heard great things about them.
+
+The whole thing sounded almost too good to be true, so joining the company was
+an easy decision.
+
+#### What is your favorite language?
+
+I don’t have many favorite things, especially when it comes to technology.
+After going through several cycles of falling in love with a
+language/framework/tool before becoming annoyed by its shortcomings and
+limitations, I made peace with the fact that everything’s a tradeoff. I’ll
+generally choose a language depending on:
+- the problem at hand
+- its current position in the aforementioned cycle :)
+
+With that said, I'm currently most excited about properly learning Haskell. It
+always looked interesting and exotic. I did get a few chances to use
+the language through university courses and on my own, but I've never really had an excuse to
+use it in my day-to-day.
+
+## What are you most excited about in Wasp?
+
+From a developer’s perspective, I’m definitely most excited about designing the
+DSL and working on the compiler.
+
+From a product standpoint, I love the idea of never again having to implement
+an authentication system.
+
+#### What’s a feature or project you’re most proud of that you worked on in the past three months?
+
+[Definitely this thing](https://www.youtube.com/watch?v=dQw4w9WgXcQ).
+
+#### How did you start coding?
+
+No exciting story to tell here, I’m afraid.
+
+I enrolled in computer science college and first started programming as a part
+of my regular “Introduction to programming” course.
+
+Before then, I didn’t really know what programming was. In fact, I  wanted to
+study electrical engineering. Unluckily for most but luckily for me, our
+college does not let you choose your major (i.e., electrical engineering or
+computing) right away. Instead, they make you go through a common year of
+universal (and some not-so-universal) courses, only allowing you to specialize
+at the start of your second year. Thanks to this quirk, I had a chance to
+encounter programming, change my mind, and enroll in computer science.
+
+#### Your dev setup?
+
+As far as hardware goes, it’s pretty basic. I use a regular laptop that’s
+starting to show signs of age (ASUS N550JX), a cheap Lenovo mouse, and my
+laptop’s integrated keyboard.
+
+The software setup is a bit more interesting. I’m usually on Manjaro Linux with
+the i3 tiling window manager. I jump between editors depending on what I’m
+doing. It’s usually either VS Code, Vim, or one of JetBrains IDEs, but I use
+Vim emulation no matter where I am. My preferred shell is zsh (sorry Frane).
+
+#### `camelCase` or `snake_case`?
+
+NoPreference
```

Comment:
lol

Reply:
you should have written `no_Preference` for ultimate heresy :D